templates/javascript/webcrypto.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
```

Comment:
I wonder why this ended up in the transpiled JavaScript... `tsc` should only prepend `"use strict"` for non-ES6 targets.

From the project root, can you try running
```bash
npm i && npm run transpile && npm run lint
```
and seeing if you get a JS output without `"use strict"` prepended?

Reply:
I still got "use strict"

Reply:
I've just removed this manually. I deleted existing js files and re ran "npm transpile" to see if my environment was prepending this to all generated js files. Nope, just this one. Also checked my webcrypto.toml file to see if there was anything funky- it matches the other metadata files pretty closely.